examples/core/cubemap/app.js

```diff
@@ -221,6 +221,6 @@ function drawTexture({ctx, sign, axis, size}) {
 export default animationLoop;
 
 /* global window */
-if (!window.website) {
+if (typeof window !== 'undefined' && !window.website) {
```

Comment:
Super nit: Create a util method given this is used in multiple places?

Reply:
This is per example as they need to work independently.